src/pages/community/foundation/local-initiative.mdx

```diff
@@ -0,0 +1,136 @@
+
+- All official events and related channels will be governed by the
+  **[GraphQL Code of Conduct](https://graphql.org/codeofconduct/)**.
+- Official communications will comply with the
+  **[GraphQL Brand Guidelines](https://graphql.org/brand/)**.
+- The Local accepts **GraphQL TSC oversight**:
+  - At least one
+    [GraphQL Technical Steering Committee (TSC) member](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#tsc-members)
+    must have administrator access to the Local’s online platforms.
+  - In case of a serious Code of Conduct breach, an organizer may be replaced.
+- The Local has 1 or more organizer from the area. (We're pretty flexible on the
+  definition of "from the area", just be reasonable.)
+- Each official event will have at least least one **organizer present**.
+- Official events will be held in an appropriate **professional setting**.
+- Official events will feature at least one GraphQL-related talk (the format can
+  vary: lecture, lightning talk, panel, AMA, etc).
+- These **talks will be recorded** and made freely available online.
+- Official events will be be **announced at least two weeks in advance**.
+  Popular ticketing platforms include Eventbrite, Guild.host, and Meetup.com.
+- Attending official events will be **affordable**.
+  - If not free, attendance fees must not exceed USD $10 or local equivalent
+    without explicit approval from the GraphQL Marketing and Content committee.
+    Events charging fees waive their entitlement to GraphQL Foundation
+    sponsorship.
+- Organizers will **report back on attendance**, general feedback, and any
+  suggestions for future events in a timely manner.
+
+Groups may of course organize other events, e.g. a summer barbeque, but these
+are currently out-of-scope of GraphQL Foundation support. These other events
+shall not be referred to as "official", but are still encouraged to follow the
+guidelines that are applicable (in particular, the Code of Conduct and Brand
+Guidelines must still be honoured).
+
+## Support
+
+Each approved Local is entitled to request the following support for each
+event:
+
+- **Sponsorship** up to USD $200 per event (max 1 per quarter) for approved
+  costs (see below). More sponsorship might be available directly from
+  individual foundation members, see details below.
+- **Swag bundle**\* to distribute to speakers, attendees, or organizers (max 1
+  per quarter).
+- **Google Slides templates** for:
+  - Code of Conduct
+  - Call for Papers (CFP)
+  - Sponsors
+  - Schedule
+  - Speaker/talk introduction
+- **Promotion** through GraphQL’s Twitter and LinkedIn channels.
+- **Custom artwork** adhering to the GraphQL Brand Guidelines; including
+  avatars, event banners, and slide templates. \*\*
+
+\* The content of swag bundles may change from time to time. At time of writing
+they are likely to consist of 2-3 t-shirts (sizes to be chosen by organizer),
+stickers, pin-badges, and a GraphQL bow-tie!
```

Comment:
@jorydotcom what's the latest on this? I wrote it from memory... and I don't have the best memory! :grimacing:

Reply:
We will give the meetup organizer a code, and right now the bundle is set up to be 3 t-shirts, 100 stickers, 10 enamel pins and 1 bow tie! I like keeping it a bit more generic now, though, as inventory may change